tmt/schemas/discover/fmf.yaml

```diff
@@ -95,5 +95,10 @@ properties:
   where:
     $ref: "/schemas/common#/definitions/where"
 
+  adjust-tests:
+    type: array
+    items:
+      type: object
```

Comment:
Shouldn't we re-use already the schema already established for `adjust`, i.e. `$ref: "/schemas/core#/definitions/adjust"`? Or, if `when` & `description` are undersirable, at least support the "not a list, but an object" way of values accepted by tmt (`prepare` can be a phase or a list of phases, `require` a package name or a list of package names, and so on). Limiting this key to a list only is well-aligned with my view, but tmt is historically more liberal, so I guess it's too late to be strict now in one case only.

Reply:
attempt in 3f12f70e